dao/source_dao.go

```diff
@@ -61,6 +61,24 @@ func (s *SourceDaoImpl) List(limit, offset int, filters []util.Filter) ([]m.Sour
 	return sources, count, result.Error
 }
 
+func (s *SourceDaoImpl) ListInternal(limit, offset int, filters []util.Filter) ([]m.Source, int64, error) {
+	query := DB.Table("sources").Select("id", "availability_status", "tenant_id")
```

Comment:
1. Why only 3 fields ?
2. We prefer (at least as we are using it everywhere) `DB.Model( &m.Source{})` instead `Table..`

Reply:
Yeah - since sources-monitor only requires those three fields and this was previously undocumented we're going to lock it down to only the 3 necessary fields, we can always expose more if needbe!

Reply:
I changed it to `DB.Model(&m.Source{})` but kept the `.Select` part, so we don't fetch more columns than necessary! Thanks for pointing it out.